app/assets/stylesheets/building_blocks.scss

```diff
@@ -0,0 +1,83 @@
+html,
+body {
+  background: #FAFAFA;
+}
+
+.card {
+  background: white;
+  border-radius: 2px;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+  color: #272727;
+  font-family: 'Roboto', sans-serif;
+  overflow: hidden;
+  .title {
+    font-size: 1.5rem;
+    font-weight: 300;
+    line-height: 3rem;
+  }
+
+  .content {
+    border-radius: 0 0 2px 2px;
+    font-weight: 300;
+    padding: 1.3rem;
+  }
+
+  p {
+    margin: 0;
+  }
+
+  .action {
+    border-top: 1px solid rgba(160, 160, 160, 0.2);
+    padding: 1.3rem;
+  }
+
+  a {
+    color: #ffab40;
```

Comment:
Color literals like `#ffab40` should only be used in variable declarations; they should be referred to via variable everywhere else.